hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -54,11 +53,6 @@ protected ServiceName getParentServiceName(PathAddress parentAddress) {
     }
 
     @Override
-    protected boolean isResourceServiceRestartAllowed(OperationContext context, ServiceController<?> service) {
```

Comment:
OK, the latest revision changes it around such that restarts kick in automatically and are deferred only if the operation explicitly disallows the restart in a header.  So basically, the new UpdateCollectionsIntervalCommand defers the restart, anything else would restart automatically. Also, I updated with the EAP6 fix, thanks @josejulio .